Answers.md

```diff
@@ -1,3 +1,15 @@
 ### Your responses to the short answer questions should be laid out here using markdown.
 
 For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+## 1. Containers are faster and use less resources than virtual machines.
+
+## 2. It is the server port that the App is running on
+
+## 3. Kubernetes simplifies the process to deploy applications
+
+## 4. I have no idea...
+
+## 5. Vertical scaling can essentially resize your server with no change to your code. Horizontal scaling affords the ability to scale wider to deal with traffic.
+
+## 6. The auto scaling is the main difference 
```

Comment:
More importantly, performance is improved because with the free tier, each container sleeps after 30 minutes of inactivity, which means that the first new request that comes in to a container that is sleeping doesn't receive a response as quickly.